apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.html.ts

```diff
@@ -160,23 +160,16 @@ function ConflictGradingJobModal({
               applied. Please review the feedback below and select how you would like to proceed.
             </div>
             <div class="row mb-2">
-              <div class="col-6">
+              <div class="col-lg-6 col-md-12">
```

Comment:
I think `col-md-12` isn't strictly necessary, but if you want to make explicit what happens on smaller viewports, I'd write this as `col-12 col-lg-6` to emphasize that it defaults to 12 on smaller viewports (not just md) but shrinks to 6 on large.